### PR TITLE
fix(Menu): Define color on menu item

### LINKED ISSUE #683: Menu item text color is not set
After updating iTwinUI react from 1.23.2 to 1.37.1 menu item text color is not explicitly set. As we were wraping menu item with anchor, now we need to explicitly override anchor text color. Before: 
image
After:
image

### PR DESCRIPTION
**Pull request by FlyersPh9**
- Define `color` on menu item.

Tested by wrapping menu item in an `<a>`.  Closes #683.

### CODE REVIEW

**Comment by bentleyvk** [comment]
If you wrapped `li` with an `a` then it is a bad test because it is an invalid HTML structure. `ul`/`ol` can only have `li` elements as direct children. Can you try using `a` inside `li` and tell whether your fix still works.

File: packages/itwinui-css/src/menu/menu.scss
Comment (mayank99) [inline packages/itwinui-css/src/menu/menu.scss:44]: This variable and the themed mixin are both removed in v1, so it doesn't make sense to release this fix to `main` branch, especially since it's not urgent

**Review by mayank99 — CHANGES_REQUESTED** [review]

File: packages/itwinui-css/src/menu/menu.scss
Comment (elephantcatdog) [reply packages/itwinui-css/src/menu/menu.scss:44]: Jon/Adam should check that the issue is fixed in v1, and if it is then we can label the issue as ready for release and not merge this.

Comment (adamhock) [reply packages/itwinui-css/src/menu/menu.scss:44]: I agree with Mayank that this issue should be addressed after v1 is merged into `main`

Comment (veekeys) [reply packages/itwinui-css/src/menu/menu.scss:44]: So use css variable and merge? Its new addition, not something we break.

Comment (adamhock) [reply packages/itwinui-css/src/menu/menu.scss]: This has been changed

Comment (mayank99) [reply packages/itwinui-css/src/menu/menu.scss]: there is no `iui-text-color` in variables package so this would still cause conflicts

**Review by mayank99 — APPROVED** [review]

**Comment by veekeys** [comment]
Anything else left in here before we merge? @adamhock 
anchor issue we could address later on, right @mayank99  ?

**Pull request merged**
